pages/search/search.jsx

```diff
@@ -126,7 +126,7 @@ class Search extends React.Component {
           placeholder="Search keywords, people, tags..."
           className="form-control"
         />
-        <button className="btn search-submit search small"></button>
+        <span className="search-glyph-container search-glyph"></span>
```

Comment:
I still find it odd to have these two classes on the same element but I guess that's just to align with how other existing glyphs are set up. Hopefully one day we can clean this up... (tbh i think the glyphs SCSS code from Foundation site is kinda outdated and needs to be updated)

Reply:
Yeah, I agree this one is a bit weird. Let me know if you have any ideas on how to un-weird it 😅

Reply:
Feel free to open a ticket for the glyphs clean up, I wouldn't mind it tbh 😸